Jenkinsfile

```diff
@@ -1,22 +1,22 @@
 pipeline {
+    environment {
+        ZENDESK_JAVA_CLIENT_TEST_URL             = credentials('ZENDESK_JAVA_CLIENT_TEST_URL')
```

Comment:
Need not be secret I guess?

Reply:
yes I prefer, it's our cloudbees sandbox env for now